docs/content/best-practices/running-in-production.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-title: Production Best Practices
+title: Running OpenFGA in Production
 description: Best Practices of Running OpenFGA in a Production Environment
-sidebar_position: 8
-slug: /getting-started/running-in-production
+slug: /best-practices/running-in-production
```

Comment:
Do you think we can keep the slug what it was for now for SEO reasons and we can move it later?

Otherwise, we can move it now - but will need to setup a redirect for it

Reply:
If we keep the slug, can we have in the same place in the sidebar?

Reply:
@rhamzeh if we keep the slug, do we need to change all the internal links again?

Reply:
Nope! Internal links are based on file name (if you link using .mdx)

But let's change the slug and will add the redirect

Reply:
Ok! Can you approve so I can merge?